src/scripts/accesskit/disable_gifs.js

```diff
@@ -14,24 +77,19 @@ const pauseGif = function (gifElement) {
     canvas.width = image.naturalWidth;
     canvas.height = image.naturalHeight;
     canvas.className = gifElement.className;
-    canvas.classList.add('xkit-paused-gif');
+    canvas.classList.add(canvasClass);
     canvas.getContext('2d').drawImage(image, 0, 0);
-
-    const gifLabel = document.createElement('p');
-    gifLabel.className = gifElement.clientWidth && gifElement.clientWidth < 150
-      ? 'xkit-paused-gif-label mini'
-      : 'xkit-paused-gif-label';
-
-    gifElement.parentNode.append(canvas, gifLabel);
+    gifElement.parentNode.append(canvas);
+    addLabel(gifElement);
```

Comment:
I'm pretty sure these two lines of code just need to be the other way around. We're using element insertion order to determine the rendering layer to avoid the need to set `z-index` and potentially fighting with Tumblr's Z indices.
```suggestion
    addLabel(gifElement);
    gifElement.parentNode.append(canvas);
```